### ISSUE TITLE
How to hide AdRotator in case no ads were shown

### ISSUE THREAD

**Issue by jack2060** [Adrotator/AdrotatorV2#57]
**Apologies, if this is the wrong forum, i could not find any discussion page anywhere.**

My assumption is , that AdRotator collapses automatically if there are no ads available, but that is not what is happening and  i am left with empty space. Is there an event or a way to check if the Ad was indeed loaded successfully?

**Comment by gintechsystems** [comment]
I agree there should be a way to detect this without having to go deep into the adrotator code.

**Comment by SimonDarksideJ** [comment]
There is an "AdLoaded" event, but you are correct. If no ads could be loaded, it should disable it'self and hide.

**Comment by jack2060** [comment]
I posted my hack at [Stackoverflow](http://stackoverflow.com/questions/29068190/how-to-detect-if-adrotator-v2-1-is-showing-an-ad-or-an-error-occurred)

I am still plagued with this issue. https://github.com/Adrotator/AdrotatorV2/issues/34
